routes/web.php

```diff
@@ -353,12 +353,17 @@ function (): void {
                             ->middleware('can:view,application')
                             ->name('manager.applications.show');
 
-                        /* View Applicant Profile */
-                        Route::get('jobs/{jobPoster}/applicants/{applicant}', 'ApplicantProfileController@show')
+                        /* View Applicant Profile with Job */
+                        Route::get('jobs/applicants/{applicant}', 'ApplicantProfileController@showWithJob')
                             ->middleware('can:manage,jobPoster')
                             ->middleware('can:view,applicant')
                             ->name('manager.applicants.show');
 
+                        /* View Applicant Profile */
+                        Route::get('applicants/{applicant}', 'ApplicantProfileController@profile')
+                            ->middleware('can:view,applicant')
+                            ->name('manager.applicants.profile');
```

Comment:
Maybe we should just move this route to the admin section, since they're the only user who's expected to use it.

Reply:
Ya I wasn't sure about which portal to put it. I though manager since it could be used by both admins, and managers in the future if need be.